components/fieldgroups/plan/healthcare-plan-details.schema.json

```diff
@@ -64,7 +64,8 @@
               "title": "Affiliations",
               "items": {
                 "type": "string"
-              }
+              },
+              "description": "."
```

Comment:
Description looks blank- should be: Patient medical plan details such as network, type, and active status.